internal/cis/client_ver_1.go

```diff
@@ -39,7 +38,6 @@ func NewClientVer1(ctx context.Context, config Config, log logrus.FieldLogger) *
 	return &ClientVer1{
 		httpClient: httpClientOAuth,
 		config:     config,
-		log:        log.WithField("client", "CIS-1.0"),
```

Comment:
Client version won't be visible in logs. This may cause trouble while debugging issues.